app/manifest/v2/firefox.json

```diff
@@ -4,20 +4,5 @@
       "id": "webextension@metamask.io",
       "strict_min_version": "78.0"
     }
-  },
-  "permissions": [
-    "storage",
-    "unlimitedStorage",
-    "clipboardWrite",
-    "http://localhost:8545/",
-    "https://*.infura.io/",
-    "https://*.codefi.network/",
-    "https://chainid.network/chains.json",
-    "https://lattice.gridplus.io/*",
-    "activeTab",
-    "tabs",
```

Comment:
we have `tabs` only unique firefox so fix this https://github.com/MetaMask/metamask-extension/pull/18218.